collectors/mimecast/collector.js

```diff
@@ -0,0 +1,204 @@
+/* -----------------------------------------------------------------------------
+ * @copyright (C) 2020, Alert Logic, Inc
+ * @doc
+ *
+ * mimecast class.
+ *
+ * @end
+ * -----------------------------------------------------------------------------
+ */
+'use strict';
+
+const moment = require('moment');
+const PawsCollector = require('@alertlogic/paws-collector').PawsCollector;
+const parse = require('@alertlogic/al-collector-js').Parse;
+const calcNextCollectionInterval = require('@alertlogic/paws-collector').calcNextCollectionInterval;
+const packageJson = require('./package.json');
+const utils = require("./utils");
+
+
+let typeIdPaths = [];
+
+let tsPaths = [];
+
+const Siem_Logs = 'SiemLogs';
+
+
+class MimecastCollector extends PawsCollector {
+    constructor(context, creds) {
+        super(context, creds, packageJson.version);
+    }
+
+    pawsInitCollectionState(event, callback) {
+        const startTs = process.env.paws_collection_start_ts ?
+            process.env.paws_collection_start_ts :
+            moment().toISOString();
+        const endTs = moment(startTs).add(this.pollInterval, 'seconds').toISOString();
+
+        const applicationNames = JSON.parse(process.env.collector_streams);
```

Comment:
Could you please send a separate PR to update collector [template](https://github.com/alertlogic/paws-collector/tree/master/collectors/template) with stream related changes.

Reply:
I'll do